### ISSUE TITLE
Using localization for client side (one-page checkout)

### ISSUE THREAD

**Issue by mariannk** [nopSolutions/nopCommerce#676]
Most of the code that's there on client side includes hard coded messages in English language. For localization purposes, they should be moved to a separate javascript object through a view and then that object should be used inside scripts. 

That I believe is a nice way to avoid hardcoding any English literals in the script files. 

For example in public.onepagecheckout.js on line 333 we see something like this 

```
validate: function () {
 var methods = document.getElementsByName('paymentmethod');
    if (methods.length == 0) {
        alert('Your order cannot be completed at this time as there is no payment methods available for it.');
        return false;
    }
```

This can easily be replaced by 

```
validate: function () {
 var methods = document.getElementsByName('paymentmethod');
    if (methods.length == 0) {
        alert(localized_data.NoPaymentMethodsError);
        return false;
    }
```

Source: http://www.nopcommerce.com/boards/t/35522/using-localization-for-client-side.aspx
See also: https://www.nopcommerce.com/boards/t/68056/english-only-text-no-localization-possible.aspx

**Comment by cnEUL** [comment]
It would be very nice if also RoxyFilemanager, I think that's Java too, could be localizable.

**Comment by DmitriyKulagin** [comment]
Closed #676
